libs/designsystem/src/lib/components/modal/services/action-sheet.helper.spec.ts

```diff
@@ -2,6 +2,8 @@ import { Component } from '@angular/core';
 import { IonicModule, ModalController as IonicModalController } from '@ionic/angular';
```

Comment:
```suggestion
import { ModalController as IonicModalController } from '@ionic/angular';
```

Consider removing unused import 😀

Reply:
Fixed in bc24f2804b8bc3bb07d8c74b668286fbcff9aa98